---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "From reads to insight: A hitchhiker's guide to ATAC-seq data analysis"
subtitle: ''
summary: ''
authors:
- Feng Yan
- David R. Powell
- David J. Curtis
- Nicholas C. Wong
tags: []
categories: []
date: '2020-12-01'
lastmod: 2021-11-24T10:00:18+11:00
featured: true
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false
  alt_text: "Roadmap of a typical ATAC-seq analysis. "

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-11-23T23:00:18.211208Z'
publication_types:
- '0'
abstract: Assay of Transposase Accessible Chromatin sequencing (ATAC-seq) is widely
  used in studying chromatin biology, but a comprehensive review of the analysis tools
  has not been completed yet. Here, we discuss the major steps in ATAC-seq data analysis,
  including pre-analysis (quality check and alignment), core analysis (peak calling),
  and advanced analysis (peak differential analysis and annotation, motif enrichment,
  footprinting, and nucleosome position analysis). We also review the reconstruction
  of transcriptional regulatory networks with multiomics data and highlight the current
  challenges of each step. Finally, we describe the potential of single-cell ATAC-seq
  and highlight the necessity of developing ATAC-seq specific analysis tools to obtain
  biologically meaningful insights.
publication: '*Genome Biology*'
url_pdf: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1929-3
doi: 10.1186/s13059-020-1929-3
---
